src/dom_components/model/ComponentImage.js

```diff
@@ -8,6 +8,7 @@ module.exports = Component.extend({
     src: '',
     void: 1,
     droppable: 0,
+    editable: 1,
```

Comment:
I used 1 here for consistency with the other properties. Just curious, but is there a reason that these are numeric instead of boolean? I know that 0 evaluates to false and 1 evaluates to true, so functionally it's fine...is it just personal preference?

Reply:
yeah, it's totally personal, using 1 character instead of 4/5 for the same functionality makes me feel better 😄